addons/binding/org.openhab.binding.ihc/build.properties

```diff
@@ -0,0 +1,10 @@
+source..=src/main/java/
+output..=target/classes
+bin.includes=META-INF/,\
+             .,\
+             OSGI-INF/,\
+             ESH-INF/,\
+             about.html,\
+             lib/commons-logging-1.2.jar,\
+             lib/org.apache.httpcomponents.httpclient_4.4.1.jar,\
```

Comment:
Any reason the standard http service within openHAB wasn't used?

Reply:
I changed http client in ihc 1 binding many years ago because of multiple problems (can't remember anymore any specific reasons). Anyhow version 1 binding has been rock solid so far, so I like to use it for v2 binding as well.